### DISCUSSION TITLE
关于Jellyfin自动更新

### DISCUSSION THREAD

**Discussion by brainiac19** [monlor/docker-xiaoya#156] [Q&A]
看了下metadata的代码，好像只有emby设置了cron定时更新？不是很明白emby和jellyfin在这的区别，求教。

**Comment by monlor** [comment]
jellyfin不是小雅维护的，没有提供每日数据的更新，所以只有emby才能每天更新媒体数据

**Reply by brainiac19** [reply]
> jellyfin不是小雅维护的，没有提供每日数据的更新，所以只有emby才能每天更新媒体数据

明白了，但又想到了几个问题。
首先是首推xiaoya+emby在我看来有些反直觉，毕竟emby很多功能都有paywall，这样的推荐组合背后有什么原因吗，还是说维护者乐意用emby而已？
其次是目前jellyfin的元数据有一个大概的更新频率可以告知我吗？我并不追求当天更新，想考虑一下是否要继续部署jellyfin，毕竟有我更熟悉的客户端。

**Reply by monlor** [reply]
我其实之前也偏向使用jellyfin，但是实际使用下来jellyfin会更占用内存。主要原因是小雅偏向emby，他只维护emby的媒体数据。另外jellyfin的媒体数据不会更新。

**Reply by brainiac19** [reply]
> 我其实之前也偏向使用jellyfin，但是实际使用下来jellyfin会更占用内存。主要原因是小雅偏向emby，他只维护emby的媒体数据。另外jellyfin的媒体数据不会更新。

明白了，感谢解答。

**Reply by brainiac19** [reply]
> 我其实之前也偏向使用jellyfin，但是实际使用下来jellyfin会更占用内存。主要原因是小雅偏向emby，他只维护emby的媒体数据。另外jellyfin的媒体数据不会更新。

还想到个问题，emby每日更新意思是每天都要下载30多g数据吗？

**Reply by monlor** [reply]
不是，每日更新的数据和初始化下载的数据不在一个地方

**Reply by brainiac19** [reply]
好的 那还是部署emby吧 谢谢作者解答

**Comment by brainiac19** [comment]
作者我又来了，还是关于媒体播放器及数据源的问题，就不新开discussion了。
部署emby后发现少了好多内容，于是手动去运行solid.py，把内容补上了。之后尝试在网页播放，提示没有兼容的流。于是在客户端播放，提示要premiere。我注意到爬虫下载的.nfo, .jpg, .strm其实都是jellyfin也支持的文件，这是否意味着我可以部署jellyfin后也使用相同的爬虫来更新内容？
